plugins/reporters/cyclonedx/src/main/kotlin/CycloneDxReporter.kt

```diff
@@ -38,7 +38,6 @@ import org.cyclonedx.model.Metadata
 import org.cyclonedx.model.license.Expression
 import org.cyclonedx.model.metadata.ToolInformation
 
-import org.ossreviewtoolkit.model.FileFormat
 import org.ossreviewtoolkit.model.Identifier
 import org.ossreviewtoolkit.model.LicenseSource
 import org.ossreviewtoolkit.model.Package
```

Comment:
commit-msg: Can you elaborate what  "This is a preparation for further cleanups." referrs to?

(I don't understand why this change is done)

Reply:
This allows the last "Remove the unused XML mapper" commit to be made. Otherwise `FileFormat.XML` would depend on Jackson's XML mapper.